lighthouse-core/formatters/partials/url-list.html

```diff
@@ -1,33 +1,27 @@
 <style>
-  .http-resources {
-    font-size: 14px;
-  }
-  .http-resource__url {
-    margin-right: 8px;
-  }
   .http-resource__protocol,
   .http-resource__code {
-    color: #999;
+    color: var(--secondary-text-color);
   }
   .http-resource__code {
     text-overflow: ellipsis;
     overflow: hidden;
   }
 </style>
 
-<div>
-  <details class="http-resources">
-    <summary>URLs</summary>
-    {{#each this}}
-      <div class="http-resource">
-        <span class="http-resource__url">{{this.url}}</span>
-        {{#if this.label}}
-          <span class="http-resource__protocol">({{this.label}})</span>
-        {{/if}}
-        {{#if this.code}}
-          <pre class="http-resource__code">{{this.code}}</pre>
-        {{/if}}
-      </div>
-    {{/each}}
-  </details>
-</div>
+<details class="subitem__details">
+  <summary class="subitem__detail">URLs</summary>
+  <ul class="subitem__details">
+  {{#each this}}
+    <li class="subitem__detail http-resource">
+      <span class="http-resource__url">{{this.url}}</span>
+      {{#if this.label}}
+        <span class="http-resource__protocol">({{this.label}})</span>
+      {{/if}}
+      {{#if this.code}}
+        <pre class="http-resource__code">{{this.code}}</pre>
```

Comment:
something is up with the indentation of code snippets:

<img width="469" alt="screen shot 2016-11-09 at 10 28 33 am" src="https://cloud.githubusercontent.com/assets/238208/20149689/4d0c4e1c-a667-11e6-839d-03157e7e8da8.png">

before:
<img width="342" alt="screen shot 2016-11-09 at 10 28 56 am" src="https://cloud.githubusercontent.com/assets/238208/20149699/55d30d9c-a667-11e6-8fbe-5af86ddea5d2.png">